database/coop.py

```diff
@@ -121,6 +121,27 @@ def book(self, tag, values):
         Coop.save_json()
         return ret
 
+    def remove(self, values):
+        ret = ""
+        for val in values:
+            if val in self.leyline:
+                self.leyline.remove(val)
+            elif val in self.weapon:
+                self.weapon.remove(val)
+            elif val in self.talent:
+                self.talent.remove(val)
+            elif val in self.artifact:
+                self.artifact.remove(val)
+            elif val in self.trounce:
+                self.trounce.remove(val)
+            elif val in self.world_boss:
+                self.world_boss.remove(val)
+            else:
```

Comment:
Maybe we can declare a collection or iterable class for all of these lists? The collection class could have methods to iterate and check for all of this stuff, and also add options something like the one you did in `dropdown_edit`